app/forms/user_piv_cac_setup_form.rb

```diff
@@ -43,7 +48,7 @@ def piv_cac_not_already_associated
     self.x509_dn_uuid = @data['uuid']
     self.x509_dn = @data['subject']
     self.x509_issuer = @data['issuer']
-    if Db::PivCacConfiguration.find_user_by_x509(x509_dn_uuid)
+    if PivCacConfiguration.find_by(x509_dn_uuid: x509_dn_uuid)&.user
```

Comment:
I'm not sure if we have orphaned `PivCacConfiguration` rows, but the uniqueness is on `x509_dn_uuid` so even if the `PivCacConfiguration` row *was* an orphan, the uniqueness would prevent creation of a new record, so we might as well show an error.

We could even simplify this to an `exists?` since we don't need to load the whole record

```suggestion
    if PivCacConfiguration.exists?(x509_dn_uuid: x509_dn_uuid)
```